lib/layers/presentation/sing_in_page/widgets/user_picker.dart

```diff
@@ -0,0 +1,81 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:wot_statistic/common/theme/text_styles.dart';
+import 'package:wot_statistic/layers/presentation/sing_in_page/bloc/sing_in_cubit.dart';
+
+import '../../../../common/constants/constants.dart';
+
+class UserPicker extends StatelessWidget {
+  const UserPicker({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final Color _onPrimary = Theme.of(context).colorScheme.onPrimary;
+    final Color _secondary = Theme.of(context).colorScheme.secondary;
```

Comment:
Please choose one naming style in color scheme. onPrimary nad onSecondary, or primary -- secondary